### PR TITLE
Updated documantation

### PR DESCRIPTION
**Pull request by MarconZet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
 # Akka Serialization Helper
 
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.virtuslab/sbt-akka-serialization-helper/badge.svg)](https://maven-badges.herokuapp.com/maven-central/cz.jirutka.rsql/rsql-parser)
```
Comment (MarconZet) [inline README.md:3]: Reminder for me: to update after #104 is merged

Comment (PawelLipski) [reply README.md:3]: `cz.jirutka.rsql`? :trollface:

File: README.md [hunk 2]
```diff
@@ -1,88 +1,130 @@
...
+
+
```
Comment (PawelLipski) [inline README.md:5]: Pls include logo while you're here... https://github.com/VirtusLab/git-machete/blob/master/README.md for a reference HTML tag FWIW

File: README.md (hunk 1 above)
Comment (PawelLipski) [reply README.md:3]: Oh, sth with 0.3.1 version, I see xD

File: README.md [hunk 3]
```diff
@@ -1,88 +1,130 @@
...
+To use the library use sbt plugin (version equals number on the maven badge):
 
 ```scala
-resolvers += "jitpack".at("https://jitpack.io")
-val commit = "master-SNAPSHOT" //name of a branch with -SNAPSHOT or raw commit hash
+addSbtPlugin("org.virtuslab.ash" % "sbt-akka-serialization-helper" % Version)
+```
+and enable it in target project:
+```scala
+enablePlugins(AkkaSerializationHelperPlugin)
```
Comment (PawelLipski) [inline README.md:18]: Maybe present in a context of `project` and its `.settings(...)`

File: README.md [hunk 4]
```diff
@@ -1,88 +1,130 @@
...
 against accidental use
 of [Java serialization](https://doc.akka.io/docs/akka/current/serialization.html#java-serialization).
 
-To use, just annotate a base trait with `@org.virtuslab.ash.SerializabilityTrait`:
+To use, annotate a base trait with `@org.virtuslab.ash.SerializabilityTrait`:
 
 ```scala
 @SerializabilityTrait
 trait MySerializable
 ```
```
Comment (PawelLipski) [inline README.md:36]: Plus maybe a small sample of a situation when an error is reported + the error itself

File: README.md [hunk 5]
```diff
@@ -1,88 +1,130 @@
...
+### 2. Dump Persistance Schema
+
+A mix of a compiler plugin and an sbt task for dumping schema
+of [akka-persistence](https://doc.akka.io/docs/akka/current/typed/persistence.html#example-and-core-api) to a
+file. It can be used for detecting accidental changes of events and states with simple `diff`.
+
+To dump persistence schema, run:
 
+```shell
+sbt ashDumpPersistenceSchema
```
Comment (PawelLipski) [inline README.md:46]: Some sample dump yaml would be useful! probably collapsed would be better, see e.g. https://raw.githubusercontent.com/VirtusLab/git-machete-intellij-plugin/develop/.github/ISSUE_TEMPLATE/bug_report.md -> `Logs`

Comment (MarconZet) [reply README.md:46]: \```yaml ...``` doesn't work inside details tag

File: README.md [hunk 6]
```diff
@@ -2,6 +2,7 @@
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.virtuslab/sbt-akka-serialization-helper/badge.svg)](https://maven-badges.herokuapp.com/maven-central/cz.jirutka.rsql/rsql-parser)
 
+![logo_ash_horizontal@4x](https://user-images.githubusercontent.com/25779550/135059025-4cfade5b-bfcb-47e8-872f-8a3d78ce0c25.png)
```
Comment (PawelLipski) [inline README.md:5]: Is this identifier stable? 🤔 I guess it should be changed into a github.com link (at least once this branch is merged)

**Review by PawelLipski — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (MarconZet) [reply README.md:5]: From my understanding it is. We will find out soon enough.

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**